### PR TITLE
Update get-to-know-hatch.md - Rewording Get to Know Hatch Documentation

### PR DESCRIPTION
**Pull request by Tyler-Bonnell**
Update/reword Hatch Features documentation

### CODE REVIEW

**Comment by lwasser** [comment]
@all-contributors please add @Tyler-Bonnell for code, review

File: tutorials/get-to-know-hatch.md [hunk 1]
```diff
@@ -113,14 +113,11 @@ and maintaining your Python package easier.
...
 :::{admonition} Comparison to other tools
 :class: tip
-[We compared hatch to several of the other popular packaging tools in the ecosystem including flit, pdm and poetry. Learn more here](package-features)
-:::
-
-[More on hatch here](hatch)
+[We compared hatch to several of the other popular packaging tools in the ecosystem including flit, pdm and poetry. Learn more here](package-features).:::
 
-A few features that hatch offers
+A few of [Hatch's features](hatch) includes:
```
Comment (lwasser) [inline tutorials/get-to-know-hatch.md:118]: great catch @Tyler-Bonnell !! just another tiny edit! 
```suggestion
A few of [Hatch's features](hatch) include:
```

**Review by lwasser — APPROVED** [review]
We can merge this when my tiny edit is merged  or fixed!

File: tutorials/get-to-know-hatch.md [hunk 2]
```diff
@@ -113,14 +113,11 @@ and maintaining your Python package easier.
 
 :::{admonition} Comparison to other tools
 :class: tip
-[We compared hatch to several of the other popular packaging tools in the ecosystem including flit, pdm and poetry. Learn more here](package-features)
-:::
-
-[More on hatch here](hatch)
+[We compared hatch to several of the other popular packaging tools in the ecosystem including flit, pdm and poetry. Learn more here](package-features).:::
```
Comment (ucodery) [inline tutorials/get-to-know-hatch.md:116]:
```suggestion
[We compared hatch to several of the other popular packaging tools in the ecosystem including flit, pdm and poetry. Learn more here.](package-features):::
```

I realize I asked for this period to move originally, but I didn't realize then it was a multi-sentence hyperlink 🙃

**Review by ucodery — APPROVED** [review]

**Pull request closed without merging**